src/middlewares/logging-middleware.ts

```diff
@@ -0,0 +1,11 @@
+import {Context} from 'koa';
+import {logger} from '../logger/logger';
+
+export const loggingMiddleware = async (ctx: Context, next: () => Promise<any>) => {
+    logger.info(`got ${ctx.request.method}: ${ctx.request.path} request from ${ctx.request.ip}`);
+    try {
+        await next();
+    } catch (err) {
```

Comment:
this is logging middleware or error handler?

Reply:
You right. It's a redundant middleware.
I added the logging into error-handler middleware